tour/authorizations.md

```diff
@@ -1,46 +1,49 @@
 ---
 title: Authorizations
 ---
-Authorizations are a set of Strings that enable a user to read protected data. A column visibility is a boolean expression 
+
+[Authorizations] are a set of Strings that enable a user to read protected data. A [ColumnVisibility] is a boolean expression
 that is evaluated using the authorizations provided by a scanner. If it evaluates to true, then the data is visible. 
```

Comment:
I think we should say that users are granted authorizations.  It would be a better lead into the example.

Reply:
Fixed in ee62cbca